File: 20221012/모범답안/20221012_08/PercentageOfLetter.js
/**
 * @param text {string}
 * @param s {string}
 * @returns {number}
 */
function solution(text, s) {
  let appearCount = 0

  for (let i = 0; i < text.length; i++) {
    if (text[i] === s) {
      appearCount++
    }
  }

  return Math.floor(appearCount / text.length * 100)
}

export default solution

File: 20221012/모범답안/20221012_08/PercentageOfLetter.py
import math


def solution(text, s):
    """
    :param text: string
    :param s: string
    :return: int
    """

    appearCount = 0

    for c in text:
        if c == s:
            appearCount += 1

    return math.floor(appearCount / len(text) * 100)
